### PR TITLE
Adds eiso.md to sourcerers

### PR DESCRIPTION
**Pull request by eiso**
A little bit about myself...

### CODE REVIEW

File: talent/sourcerers/eiso.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- If you wanted to do an investment, what company would you choose?
+  - I am all in on source{d} but if I had to invest in something else, it would probably be related to innovation in agriculture or education (the fundamentals of progress). 
+
+**Other**
+- What are your hobbies and what do you like about them?
+  - I have never been a person who had a lot of hobbies. I am happiest when I have a combination of spending time with family and a few close friends, being able to work on things I love with interesting and kind-hearted people, and finding time to be healthy and exercise.
+  - What you could call hobbies is that since a month I surf, started boxing again and do a lot of yoga. 
+  - When I am rested I enjoy reading, when I am tired I love watching TV shows and movies.
+- What was the last lie you told?
+  - Honestly, can't remember
```
Comment (marnovo) [inline talent/sourcerers/eiso.md]: Was that a lie? ;)

File: talent/sourcerers/eiso.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- Would like to share your Twitter, Instagram or Medium or any other social media account?
+  - I am only on Twitter ([@eisokant](twitter.com/eisokant)), I am not a big fan of social media (I like digital privacy).
+
+**Work**
+- What are you going to be doing at source{d}?
+  - Keep on building a great team, help set direction and always try to be clear and open in communications.
+- Why did you join source{d}?
+  - I co-founded it. Today I would join source{d} because of the people and the immense passion for programming that is shared by the team.
+- Where will you work from? 
+  - Either remote from Lisbon or from our San Francisco office, once in a while you'll see me drop by in Madrid for our gettogethers.
```
Comment (marnovo) [inline talent/sourcerers/eiso.md]: s/gettogethers/get-togethers

File: talent/sourcerers/eiso.md (hunk 1 above)
Comment (eiso) [reply talent/sourcerers/eiso.md]: Funnily enough, it wasn't.

**Comment by eiso** [comment]
No need, DCO is for code and documentation to projects which have value in their IP/copyright. I doubt my life opinions fall under that ;).

**Pull request merged**